### DISCUSSION TITLE
Book mark ideas

### DISCUSSION THREAD

**Discussion by TaharBelghitriDz** [dailydotdev/daily#713] [Feature Requests]
users can have more then one type of book mark ... let theme have some named book mark like uxui / backend / web3 and move one

**Comment by idoshamun** [comment]
Hi @TaharBelghitriDz,
Not sure I understand your idea. Can you please elaborate?

**Reply by TaharBelghitriDz** [reply]
I want to create different collections for different types of articles ... for example like youtube

**Reply by idoshamun** [reply]
Oh I see. This suggestion already exists here: https://github.com/dailydotdev/daily/discussions/520

**Comment by seeniforu** [comment]
I think we can have like if bookmark article is already readed, move to readed-bookmarks. If not keep it in unread-bookmarks.
I hope this will be helpful.
